ui/admin/tests/acceptance/host-catalogs/hosts/update-test.js

```diff
@@ -130,7 +130,7 @@ module('Acceptance | host-catalogs | hosts | update', function (hooks) {
       'The request was invalid.',
     );
     assert.ok(
-      find('.rose-form-error-message').textContent.trim(),
+      find('.hds-form-error__message').textContent.trim(),
```

Comment:
Can we start using data-attrs here like @cameronperera mentioned in tech time?

Reply:
@DhariniJeeva Absolutely!